crates/turborepo-lib/src/commands/login.rs

```diff
@@ -19,6 +19,21 @@ const DEFAULT_PORT: u16 = 9789;
 const DEFAULT_SSO_PROVIDER: &str = "SAML/OIDC Single Sign-On";
 
 pub async fn sso_login(base: &mut CommandBase, sso_team: &str) -> Result<()> {
+    if let Some(token) = base.user_config()?.token() {
+        // Try to fetch the user with the token we found to see if we get a good
+        // response back.
+        // TODO: Write response to disk close to / related to token.
+        match base.api_client()?.get_user(token).await {
```

Comment:
`sso_login` can't just check `get_user()`, I think it needs to check `get_teams()` and match the passed `sso_team` arg to it to make sure the current token on disk actually has access to that team